litmus-portal/frontend/cypress/component/stepper.spec.tsx

```diff
@@ -8,13 +9,20 @@ import configureStore from '../../src/redux/configureStore';
 
 const { store } = configureStore();
 
-// Test Suite - Stepper Labels are present
-describe('Input Data is present', () => {
-  const wrapper = (
+const client = new ApolloClient({
+  cache: new InMemoryCache(),
```

Comment:
Why do you need Apollo for a stepper unit test?

Reply:
Already resolved in your previous review, Told you that right now the stepper is using usemutation hook, which needs apollo to be wrapped around it.
I know, this is a bit off for unit tests, for now this is the work around for this.I will fix it when i find something better than this.